.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-arm64
```

Comment:
Saw your post on Mastodon. I think you might have to drop the 64. Just arm. It's cleaner. (Jokes aside this should be a lint / warn)

Reply:
🫡 you're a nice person, thanks!

Reply:
Oh, and I understood your reference. Quite funny. 😆